pelican/__init__.py

```diff
@@ -327,6 +329,16 @@ def parse_arguments():
                         help=('Exit the program with non-zero status if any '
                               'errors/warnings encountered.'))
 
+    parser.add_argument('-l', '--listen', dest='listen', action='store_true',
+                        help='Serve content files via HTTP and port 8000.')
+
+    parser.add_argument('-p', '--port', dest='port', type=int, default=8000,
+                        help='Port to serve HTTP files at. (default: 8000)')
+
+    parser.add_argument('-b', '--bind', dest='bind', default='127.0.0.1',
+                        help='IP to bind to when serving files via HTTP '
+                        '(default: 127.0.0.1)')
+
```

Comment:
`-p/--port` and `-b/--bind` are useless without `-l/--listen`. It would be nice to warn the user if `-p` and/or `-b` used without `-l`.